content/en/docs/v3.6/op-guide/recovery.md

```diff
@@ -64,10 +137,4 @@ $ etcd \
   --listen-peer-urls http://host3:2380 &
 ```
 
-Now the restored etcd cluster should be available and serving the keyspace given by the snapshot.
-
-## Restoring a cluster from membership mis-reconfiguration with wrong URLs
```

Comment:
Suggest to keep this section. We should clearly call out the workaround `--force-new-cluster` although it isn't recommended.

Reply:
I have merged the useful parts into `Restoring with updated membership` above, not sure the issue links for 3.2 are very relevant nowadays. Hence the removal.